services/web/server/src/simcore_service_webserver/director_v2_core_dynamic_services.py

```diff
@@ -117,44 +123,89 @@ async def run_dynamic_service(
 
 @log_decorator(logger=log)
 async def stop_dynamic_service(
-    app: web.Application, service_uuid: str, save_state: bool = True
+    app: web.Application,
+    service_uuid: str,
+    save_state: bool = True,
+    progress: Optional[ProgressBarData] = None,
```

Comment:
and must it be optional? optional arguments are for testing. do we need it like that?